### ISSUE TITLE
No 'Access-Control-Allow-Origin' header is present on the requested resource. (urgent help needed)

### ISSUE THREAD

**Issue by christoph-LSN** [open-sdg/open-sdg#1824]
@MoBosse @brockfanning @LucyGwilliamAdmin @otis-bath 
He guys, it's a little urgent. I have to go online next wednesday... The site works on the custom domain but no data is shown.
What I have done wrong?

**Please complete the following information** 
Site repository URL: [(https://github.com/christoph-LSN/MT_Site/)](https://github.com/christoph-LSN/MT_Site/)
Data repository URL: [(https://github.com/christoph-LSN/IM_Data/)](https://github.com/christoph-LSN/IM_Data/)

[Production Organization](https://github.com/MT-production)
[Production Site](https://github.com/MT-production/MT-production.github.io)
[Production Data](https://github.com/MT-production/IM_production_data)
[Custom Domain: Integrationsmonitoring.niedersachsen.de](https://integrationsmonitoring.niedersachsen.de/)

Example Indicator: https://integrationsmonitoring.niedersachsen.de/1-1-1/

Open SDG version: [e.g. 2.1.0-dev]

**Describe the bug**
Data is not loading.

Inspecting the site shows the following error:
integrationsmonitoring.niedersachsen.de/:1 Access to XMLHttpRequest at 'https://mt-production.github.io/IM_production_data/de/comb/1-1-1.json' from origin 'https://integrationsmonitoring.niedersachsen.de' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

I can access the json file directly: https://integrationsmonitoring.niedersachsen.de/IM_production_data/de/comb/1-1-1.json

Additional Information:
Merging the site develop branch into the master branch shows a lot of errors during the html test which you can see e.g. [here](https://github.com/christoph-LSN/MT_Site/actions/runs/3205730516/jobs/5238608745)
I am not sure if this has something do do with this issue.

the depyloy to production runs without any warning or failure

Additional Information 2 (also not sure):
Taking a look at the master branche in my MT-production.github.io
this line https://github.com/MT-production/MT-production.github.io/blob/96df26444518ed0621e29b5d6a5b1909dc1caf17/1-1-1/index.html#L14
has:

`// A base URL for asynchronous calls to SDG data.
  remoteDataBaseUrl: 'https://MT-production.github.io/IM_production_data/de',`

two problems: 
1) If i try to access https://MT-production.github.io/IM_production_data/de directly nothing can be found here but here: https://MT-production.github.io/IM_production_data/ and here https://integrationsmonitoring.niedersachsen.de/IM_production_data/ as well.

2) and to my mind the remoteDataBaseUrl: should point to: https://integrationsmonitoring.niedersachsen.de/IM_production_data/de 

**Screenshots**
![image](https://user-images.githubusercontent.com/94443931/194589436-3b4f1cfa-b5d8-4c03-8e0e-9b38d19d22f3.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - Browser: [chrome]

**Comment by brockfanning** [comment]
@christoph-LSN My first thought would be to change this line: https://github.com/christoph-LSN/MT_Site/blob/develop/_data/site_config_prod.yml#L19

Instead of

```
remote_data_prefix: "https://MT-production.github.io/IM_production_data"
```

Try:

```
remote_data_prefix: "https://integrationsmonitoring.niedersachsen.de/IM_production_data"
```

**Comment by christoph-LSN** [comment]
@brockfanning Great it works! Thank you so much...
Perhaps we need to update the documentaition here:
https://github.com/open-sdg/open-sdg/edit/master/docs/hosting/production.md#L24

Nevertheless to me this behavior is some kind of strange: because adding the CNAME to MT-production.github.io hast to change the remote_data_prefix as well...

**Comment by christoph-LSN** [comment]
closed with #1825
